### PR TITLE
Add an option to specify time format

### PR DESCRIPTION
**Pull request by eltoder**
str(float) prints 16 decimal places, which is too verbose and much more than the precision in the profiles. Print 7 digits by default and allow setting the format with --time-format.

### CODE REVIEW

File: gprof2dot.py [hunk 1]
```diff
@@ -44,6 +44,7 @@
 
 
 MULTIPLICATION_SIGN = chr(0xd7)
+TIME_FORMAT = "%.7g"
```
Comment (jrfonseca) [inline gprof2dot.py:47]: Given `TIME_FORMAT` is not a constant, but rather a variable, please rename it to `timeFormat`.

**Review by jrfonseca — COMMENTED** [review]
Otherwise looks good.

I actually wonder if `--time-format`  option is worthwhile, but have no strong opinion either way.

File: gprof2dot.py (hunk 1 above)
Comment (eltoder) [reply gprof2dot.py]: done

**Comment by eltoder** [comment]
> Otherwise looks good.
> 
> I actually wonder if `--time-format` option is worthwhile, but have no strong opinion either way.

I think picking a format that pleases everyone can be hard and it's easy to make it configurable, so we can avoid arguing about it.

**Comment by eltoder** [comment]
@jrfonseca please take another look

btw, `--time-format` can also be used to address #58 -- if one wants to see units in the labels, they can add it to the format, e.g. `--time-format=%.6gs`.

**Comment by jrfonseca** [comment]
> btw, `--time-format` can also be used to address #58 -- if one wants to see units in the labels, they can add it to the format, e.g. `--time-format=%.6gs`.

Good point.

Merged.  Thanks.

**Comment by eltoder** [comment]
Thank you

**Pull request merged**